tools/release/cloudbuild.yaml

```diff
@@ -0,0 +1,119 @@
+steps:
+
+# Get the GitHub access token from Secret Manager.
+# The token is used to talk to GitHub API, specifically to create a release.
+- name: gcr.io/cloud-builders/gcloud
+  id: 'github-token'
+  entrypoint: 'bash'
+  args: 
+  - '-c'
+  - 'gcloud secrets versions access latest --secret=kaggleteam-github-access-token > /root/github_token'
+  volumes:
+  - name: 'root'
+    path: /root
+
+# Get the pypi password from Secret Manager, and create the ~/.pypirc file.
+- name: 'gcr.io/cloud-builders/gcloud'
+  id: create-credentials-pypirc
+  entrypoint: 'bash'
+  args:
+    - '-c'
+    - |
+      token_test=$(gcloud secrets versions access latest --secret=test-pypi-token)
+      token=$(gcloud secrets versions access latest --secret=pypi-token)
+      cat >~/.pypirc <<EOL
+      [distutils]
+      index-servers =
+        pypi
+        pypitest
+      [pypi]
+      repository: https://upload.pypi.org/legacy/
+      username=__token__
+      password=${token}
+      [pypitest]
+      repository: https://test.pypi.org/legacy/
+      username=__token__
+      password=${token_test}
+      EOL
+  volumes:
+  - name: 'root'
+    path: /root
+
+# Pull the repository from GitHub.
+- name: 'gcr.io/cloud-builders/git'
+  id: clone-repo
+  args:
+  - clone
+  - https://github.com/kaggle/$_GITHUB_REPOSITORY.git
+
+# The hatch docker image is built with the tools/cicd/cloudbuild.yaml build.
+- name: gcr.io/$PROJECT_ID/hatch:${_PYTHON_VERSION}
+  id: tests
+  dir: $_GITHUB_REPOSITORY
+  args:
+  - run
+  - test
+
+- name: gcr.io/$PROJECT_ID/hatch:${_PYTHON_VERSION}
+  id: lint
+  dir: $_GITHUB_REPOSITORY
+  args:
+  - run
+  - lint:all
+
+# Delete old releases if exist.
```

Comment:
Is this necessary given the `gcr.io/$PROJECT_ID/hatch` image probably doesn't have any pre-baked?